installer.sh

```diff
@@ -371,9 +372,9 @@ case "$platform" in
 				install_package_macos $pkg
 			done
 		else
-			echo "${yellow}Some dependency utils need installed though brew, which is not installed."
-			echo "If the installation is failed due to dependedncy utils not found."
-			echo "Please install it manually.${normal}"
+			echo "${yellow}Since brew is not installed, frequently-used dependencies could not be installed."
```

Comment:
should we not output here also the "affected" packages? or we need them in doc and a link, or?

Reply:
I'm not sure, that seems like to much noise. That's why I referred to the adapter documentations.
But I'm fine with both.

Reply:
I'm not sure, that seems like to much noise. That's why I referred to the adapter documentations.
But I'm fine with both.

Reply:
Ok, then we leave as is. If we get user feedback we can change it still afterwards